### PR TITLE
2 changes have been done

### PR DESCRIPTION
**Pull request by yaswanth2802**
- created the file requirements.txt which contains packages involved for jupyter notebooks .(which helps to install all the packages required by the command pip install -n requirements.txt  without running multiple number of commands.

- changed the code in digit-recognizer.ipynb by creating a different model which increased the accuracy to 0.9974 and decreased the loss to 0.0097

- to see the change in accuracy please run the modified digit-recognizer.ipynb file in your laptop.
Screenshot (876)

### CODE REVIEW

**Comment by Rivatus** [comment]
Hello @yaswanth2802 , to create a PR you must have created an issue first. 
Also there is a serious issue with your code, you trained it on complete dataset, and tested it on the same dataset, this is a bad practice and is usually a source of error as you don't know how your data will perform on unseen data. Here's your model's accuracy in testing dataset (10%): 
image

Also you have used the standard neural network model, which in comparison to CNN is not good for image identification and classification. Hope you identify your mistakes, if you still have doubts, you can further question.
